lms/djangoapps/instructor/paidcourse_enrollment_report.py

```diff
@@ -156,7 +156,8 @@ def _get_order_data(self, registration_code_redemption, course_id):
         coupon_codes = [redemption.coupon.code for redemption in coupon_redemption]
         coupon_codes = ", ".join(coupon_codes)
 
-        list_price = order_item.list_price
+        # We store the list price only when a discount has been applied, otherwise, we just use the unit_cost.
+        list_price = order_item.list_price if order_item.list_price else order_item.unit_cost
```

Comment:
Can we have a helper method on OrderItem, say get_list_price(), which does this None handling? That will help clean up some of the business logic....